packages/swapper/src/swappers/cow/getCowSwapTradeQuote/getCowSwapTradeQuote.test.ts

```diff
@@ -0,0 +1,222 @@
+import { ethereum } from '@shapeshiftoss/chain-adapters'
+import { HDWallet } from '@shapeshiftoss/hdwallet-core'
+import { Asset } from '@shapeshiftoss/types'
+import Web3 from 'web3'
+
+import { GetEthTradeQuoteInput, TradeQuote } from '../../../api'
+import { ETH, FOX, WBTC, WETH } from '../../utils/test-data/assets'
+import { CowSwapperDeps } from '../CowSwapper'
+import { cowService } from '../utils/cowService'
+import { getCowSwapTradeQuote } from './getCowSwapTradeQuote'
+
+jest.mock('@shapeshiftoss/chain-adapters')
+jest.mock('../utils/cowService')
+jest.mock('../utils/helpers/helpers', () => {
+  return {
+    getUsdRate: (_args: CowSwapperDeps, input: Asset) => {
+      if (input.assetId === WETH.assetId) {
+        return Promise.resolve('1233.65940923824103061992')
+      }
+
+      return Promise.resolve('20978.38')
+    }
+  }
+})
+
+const feeData = {
+  fast: {
+    txFee: '4080654495000000',
+    chainSpecific: {
+      gasLimit: '51630',
+      gasPrice: '79036500000',
+      maxFeePerGas: '216214758112',
+      maxPriorityFeePerGas: '2982734547'
+    }
+  }
+}
+
+const expectedApiInputWethToFox = {
```

Comment:
Add a new type/interface and type these API inputs.

Reply:
done